distutils/cygwinccompiler.py

```diff
@@ -341,7 +341,9 @@ def __init__(self, verbose=0, dry_run=0, force=0):
 
     def runtime_library_dir_option(self, dir):
         raise DistutilsPlatformError(
-            "don't know how to set runtime library search path on Windows"
+            "don't know how to set runtime library search path on Windows\n"
```

Comment:
The original message appears to be a copy/paste from line 279. Probably we'll want to consolidate the messaging (even though one is an error and one is a warning). I'll use this commit as inspiration for a more concise message.